tests/test-dirs/config/dot-merlin-reader/load-config.t

```diff
@@ -44,6 +47,10 @@ but we want to observe the stdlib being configured via the STDLIB directive.
     ],
     "stdlib": "/stdlib",
     "unit_name": null,
+    "unit_name_for": {
+      "b": "b",
```

Comment:
It seems like this should be `Prefix__b` because you set `WRAPPING_PREFIX`?  That seems like a bug - probably we don't care because we aren't going to use `WRAPPING_PREFIX`, but maybe change the test not to use it at all?